### PR TITLE
Issue0076

### PR DESCRIPTION
**Pull request by ashleymurray**
Part one of MiniTask

### CODE REVIEW

File: content/ict-security-policy.md [hunk 1]
```diff
@@ -97,7 +97,7 @@ The foundation of good Information Security is the establishment and operation o
 
 The ICT security management structure falls within the overall MoJ security management structure.
 
-The Permanent Secretary, as Accounting Officer, has overall responsibility for all aspects of security. The Departmental Security Officer (DSO) and MoJ SIRO support the Permanent Secretary by providing advice on personal, physical and information security policy and procedure. The Chief IT Security Officer (CISO) supports the DSO by providing advice on ICT security policy.
+The Permanent Secretary, as Accounting Officer, has overall responsibility for all aspects of security. The Senior Security Advisor (SSA) and MoJ SIRO support the Permanent Secretary by providing advice on personal, physical and information security policy and procedure. The Chief IT Security Officer (CISO) supports the SSA by providing advice on ICT security policy. This structure is depicted in Figure 2.
```
Comment (cybersquirrel) [inline content/ict-security-policy.md]: Chief Information Security Officer.

Comment (warmanaMOJ) [reply content/ict-security-policy.md]: We definitely avoid email addresses for people by name (anything that might be considered personal/identifying information). But email addresses for roles/teams should be OK.

Comment (cybersquirrel) [reply content/ict-security-policy.md]: Ok, that makes sense.

**Comment by warmanaMOJ** [comment]
Perfectly reasonable changes, but suggested for the wrong file. The changes have been re-submitted in PR #80 , so closing this one. Thank you.

**Pull request closed without merging**